apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/BigtableByteString.cs

```diff
@@ -0,0 +1,301 @@
+
+        /// <summary>
+        /// Converts the <see cref="BigtableByteString"/> value to a string assuming it is UTF-8 encoded.
+        /// </summary>
+        /// <param name="value">
+        /// The BigtableByteString value to convert.
+        /// </param>
+        public static explicit operator string(BigtableByteString value) => value.Value.ToStringUtf8();
+
+        /// <summary>
+        /// Converts the big-endian encoding of a signed 64-bit value to a <see cref="BigtableByteString"/> value.
+        /// </summary>
+        /// <param name="value">
+        /// The 64-bit value which should be used to initialize the byte string.
+        /// </param>
+        public static implicit operator BigtableByteString(long value) => new BigtableByteString(value);
+
+        /// <summary>
+        /// Converts the <see cref="BigtableByteString"/> value to its a signed 64-bit value assuming it is big-endian encoded.
+        /// </summary>
+        /// <param name="value">
+        /// The BigtableByteString value to convert.
+        /// </param>
+        public static explicit operator long(BigtableByteString value)
+        {
+            var bytes = value.Value.ToByteArray();
+            if (BitConverter.IsLittleEndian)
+            {
+                Array.Reverse(bytes);
+            }
+            return BitConverter.ToInt64(bytes, 0);
+        }
+
+        /// <summary>
+        /// Converts a byte array to a <see cref="BigtableByteString"/> value.
+        /// </summary>
+        /// <param name="value">
+        /// The bytes making up the byte string. May be null in which case the empty byte string is used.
+        /// </param>
+        public static implicit operator BigtableByteString(byte[] value) => new BigtableByteString(value);
+
+        /// <summary>
+        /// Converts the <see cref="BigtableByteString"/> value to its underlying byte array.
+        /// </summary>
+        /// <param name="value">
+        /// The BigtableByteString value to convert.
+        /// </param>
+        public static explicit operator byte[] (BigtableByteString value) => value.Value.ToByteArray();
+
+        /// <summary>
+        /// Converts a byte string to a <see cref="BigtableByteString"/> value.
+        /// </summary>
+        /// <param name="value">
+        /// The byte string making up the value. May be null in which case the empty byte string is used.
+        /// </param>
+        public static implicit operator BigtableByteString(ByteString value) => new BigtableByteString(value);
+
+        /// <summary>
+        /// Converts the <see cref="BigtableByteString"/> value to its underlying byte string.
+        /// </summary>
+        /// <param name="value">
+        /// The BigtableByteString value to convert.
+        /// </param>
+        public static implicit operator ByteString(BigtableByteString value) => value.Value;
```

Comment:
Is this one deliberately implicit because it's just unwrapping? That's *probably* okay...

Reply:
Yes, I figured since it's just a thin wrapper, it should be ok to convert between them automatically. However, the only oddity is that converting a null `ByteString` to `BigtableByteString` and then back would change it to `ByteString.Empty`. I guess I don't feel too strongly about it. If you think it should go back to being explicit, I will change it.

Reply:
Nope, I think that's okay.